test/test_gesvd.cc

```diff
@@ -277,6 +315,60 @@ void test_gesvd_work(Params& params, bool run)
                 printf( "ScaLAPACK not available\n" );
         #endif
     }
+
+    if (check && (wantu || wantvt)) {
+        slate::Matrix<scalar_t> Iden;
+        Iden = slate::Matrix<scalar_t>(min_mn, min_mn, nb, p, q, MPI_COMM_WORLD);
+        Iden.insertLocalTiles();
+
+        if (wantu) {
+            //==================================================
+            // Test results by checking orthogonality of U
+            //
+            //      || I - U^H U ||_1
+            //     ------------------- < tol * epsilon
+            //              N
+            //==================================================
+            slate::set( zero, one, Iden );
+            auto UH = conj_transpose( U );
+            slate::gemm( -one, UH, U, one, Iden );
+            params.ortho_U() = slate::norm( slate::Norm::One, Iden ) / n;
+            params.okay() = params.okay() && (params.ortho_U() <= tol);
+        }
+
+        if (wantvt) {
+            //==================================================
+            // Test results by checking orthogonality of VT
+            //
+            //      || I - V V^H ||_1
+            //     ------------------- < tol * epsilon
+            //              N
+            //==================================================
+            slate::set( zero, one, Iden );
+            auto V = conj_transpose( VT );
+            slate::gemm( -one, VT, V, one, Iden );
+            params.ortho_V() = slate::norm( slate::Norm::One, Iden ) / n;
+            params.okay() = params.okay() && (params.ortho_V() <= tol);
+        }
+
+        if (wantu && wantvt) {
+            //==================================================
+            // Test results by checking backwards error
+            //
+            //      || Acpy - U Sigma VT ||_1
+            //     --------------------------- < tol * epsilon
+            //            || A ||_1 * N
+            //
+            //==================================================
+            // todo:
+            slate::scale_row_col( slate::Equed::Col, Sigma, Sigma, U );
```

Comment:
Side note: perhaps this enum shouldn't be called `Equed`, since we use it in other contexts than equilibration, such as here. Maybe `slate::Scaling`? What do you think?

I'm not even sure what `Equed` means, why in LAPACK it isn't `Equil` or something. Cf. [dsysvxx.f](https://github.com/Reference-LAPACK/lapack/blob/master/SRC/dsysvxx.f)

Reply:
Right, using slate::Scaling is much better and make sense.